spec/fixtures/dataspace_migration/attention/Attention_Awareness_Dorsal_Attention_README.txt

```diff
@@ -0,0 +1,155 @@
+This Attention_Awareness_Dorsal_Attention_readme.txt file was generated on 2020-11-17 by Andrew I. Wilterson

```

Comment:
You might consider just naming this `README.txt`: I think it will then be shown when looking at the directory in github. Or there might already be a convention that dictates this pattern.